src/faebryk/libs/sexp/dataclass_sexp.py

```diff
@@ -181,6 +185,13 @@ def _decode[T](
                     if str(key) + "s" in key_fields or str(key) in key_fields:
                         ungrouped_key_values.append(val)
                         continue
+
+        # The only symbol this is currently used for is the hide symbol in schematics
```

Comment:
`... hide)` was parsed as a positional previously. I added this code to accommodate it

Reply:
Unfortunately that doesn't generalize :(
If a token is parsed as parseMaybeAbsentBool a token without braces is evaluated to default and not true. There is (at least) one hide you are dealing with that is not a parseMaybeAbsentBool and thus behaves differently. We need to give for that one a hint to the parser in some way.

Sidenote: Isn't it fine to mark it positional?

If not I would suggest making a new class (like with SymEnum).